### DISCUSSION TITLE
High CPU load from Freeplane on MacBook Pro 2020 (Intel)

### DISCUSSION THREAD

**Discussion by DanielK37** [freeplane/freeplane#1739] [Help] [answered]
Hi all,

I have a problem running Freeplane on a MacBook Pro (Intel i5 Quad-Core 2020, 16 GB RAM; Sonoma 14.4). When I initially installed it a few months ago it ran without problems. Now yesterday (and reproduced today) I experienced very high cpu load right after starting the freeplane app (actually the process shows to be using > 100% CPU). After about half a minute also the fans start blowing at highest rpm. It starts when starting the app, I do not open a file yet. Actually I am not able to open a file at all then. When I kill the process through activity monitor, everything goes back to normal, CPU goes down and fans are getting silent again.

I have another iMac (also 2020 but Intel i5 6-Core, 24 GB RAM; Sonoma 14.4) that does not experience this issue. So I expect this not to be a bug per se actually but maybe a problem of the local machine (cache or something?).

Any help is greatly appreciated.

Thanks
Daniel

**Comment by dpolivaev** [comment, marked as answer]
Try to rename or remove all user specific Freeplane configuration files and directories when no Freeplane instance is running. On my mac they are in /Users/dimitry/.freeplane/1.11.x.

**Reply by DanielK37** [reply]
Thanks, that solved it. I looked for configuration/cache files but didn't look for "dot"-files.

**Reply by dpolivaev** [reply]
Can you send me the deleted files so that I can investigate the bug?
